ideas/279-snt-voting-dapp.md

```diff
@@ -0,0 +1,106 @@
+---
+id: 279-snt-voting-dapp
+title: SNT Voting Dapp
+status: draft
+created: 2018-06-05
+category: dapp
+contributors:
+    - Barry 
+    - 3esmit
+    - rramos
+exit-criteria: yes
+success-metrics: yes
+clear-roles: yes
+future-iteration: yes
+roles-needed:
+    - QA
+    - UXR
+    - Designer
+okrs:
+    -
+    -
+---
+
+## Preamble
+
+    Idea: 279
+    Title: SNT Voting Dapp
+    Status: Draft
+    Created: 2018-05-21
+    Requires (*optional): 172-topic-democracy
+
+## Summary
+
+Provide Status users with a dApp to vote on ideas using their SNT without affecting their token balance.
+
+## Swarm Participants
+
+- Lead Contributor: @3esmit
+- Contributor: @Barry
+- Contributor: @rramos
+- Contributor: @3esmit
+- QA:
+- PM (required for user-facing): @rachelhamlin
+- UX(R) (required for user-facing swarms):
+
+## Product Overview
+
+Status.im has ideas for a DApp idea collective commons, and we want the SNT Holders to have the ability to vote on these and decide a winner
+
+
+### Requirements & Dependencies
+
+The final goal for this DApp is to be managed via Liquid Democracy. Idea #172 needs to be implemented before this iteration starts.
```

Comment:
Which iteration is this referring to?

My understanding is we can do a naive version of this without 172, so this is referring to iteration N? Or am I misunderstanding?